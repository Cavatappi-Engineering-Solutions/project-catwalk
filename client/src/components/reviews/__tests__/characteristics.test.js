/* eslint-disable no-undef */
import React from 'react'
import renderer from 'react-test-renderer'
import Characteristics from '../characteristics.jsx'

it('renders correctly', () => {
  const tree = renderer.create(<Characteristics
    name={'test'}
    characteristics={{
      Size: { id: 36850 },
      Width: { id: 36851 },
      Comfort: { id: 36852 }
    }}
  />).toJSON()
  expect(tree).toMatchSnapshot()
})
