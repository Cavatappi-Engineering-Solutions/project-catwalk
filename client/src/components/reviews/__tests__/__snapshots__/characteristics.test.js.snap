// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  id="characteristics"
>
  <div
    className="characteristic"
  >
    <span
      className="char-name"
    >
      Size
    </span>
    <div>
      <div
        className="char-bar"
      />
      <div
        className="char-marker"
        style={
          Object {
            "--marker-position": "NaN%",
          }
        }
      >
        ▼
      </div>
    </div>
    <div
      className="char-meaning"
    >
      <span
        className="char-meaning-low"
      />
      <span
        className="char-meaning-high"
      />
    </div>
  </div>
  <div
    className="characteristic"
  >
    <span
      className="char-name"
    >
      Width
    </span>
    <div>
      <div
        className="char-bar"
      />
      <div
        className="char-marker"
        style={
          Object {
            "--marker-position": "NaN%",
          }
        }
      >
        ▼
      </div>
    </div>
    <div
      className="char-meaning"
    >
      <span
        className="char-meaning-low"
      />
      <span
        className="char-meaning-high"
      />
    </div>
  </div>
  <div
    className="characteristic"
  >
    <span
      className="char-name"
    >
      Comfort
    </span>
    <div>
      <div
        className="char-bar"
      />
      <div
        className="char-marker"
        style={
          Object {
            "--marker-position": "NaN%",
          }
        }
      >
        ▼
      </div>
    </div>
    <div
      className="char-meaning"
    >
      <span
        className="char-meaning-low"
      />
      <span
        className="char-meaning-high"
      />
    </div>
  </div>
</div>
`;
